tests/conftest.py

```diff
@@ -72,305 +58,35 @@ def polarion_testsuite_properties(record_testsuite_property):
         )
 
 
-def cluster_teardown(log_level="DEBUG"):
-    log.info("Destroying the test cluster")
-    destroy_cluster(config.ENV_DATA['cluster_path'], log_level)
-    log.info("Destroying the test cluster complete")
-
-
 @pytest.fixture(scope="session", autouse=True)
 def cluster(request, log_cli_level):
+    """
+    This fixture initiates deployment for both OCP and OCS clusters.
+    Specific platform deployment classes will handle the fine details
+    of action
+    """
     log.info(f"All logs located at {log_path}")
     log.info("Running OCS basic installation")
     log.info(f"Openshift Installer will use log level: {log_cli_level}")
-    cluster_path = config.ENV_DATA['cluster_path']
-    deploy = config.RUN['cli_params']['deploy']
+
     teardown = config.RUN['cli_params']['teardown']
+    factory = dep_factory.DeploymentFactory()
+    deployer = factory.get_deployment()
+    log.info(type(deployer))
```

Comment:
Please do the message more verbose and properly format like "Deployer type: {type(deployer)}", the same bellow please

Reply:
Ok, This was my debugging purpose. Removed it.